python/edgar_filing_searcher/parsers/parser_context.py

```diff
@@ -0,0 +1,14 @@
+"""Creates a Flask SQLAlchemy app"""
+import os
+from flask import Flask
+from edgar_filing_searcher.database import db
+
+
+def create_app():
+    """This function creates a Flask SQLAlchemy app"""
```

Comment:
You should probably also include that this will not be used as a flask sqlalchemy app but instead is just used to create a context so that you can use sqlalchemy.